de.fhdo.lemma.technology.technologydsl.metamodel/model/TechnologyDefinitionModel.xcore

```diff
@@ -446,6 +446,13 @@ class DeploymentTechnology extends OperationTechnology {
         nameParts.add(name)
         nameParts.asEList
     }
+
+    /**
+     * Get the qualified name name based in the technology and deployment technology name
+     */
+    op String getQualifiedTechnologyName() {
```

Comment:
The concept already has a method called `getQualifiedNameParts()`. So, why not providing a method called `buildQualifiedName()`, which draws on `getQualifiedNameParts()` and thus keeps naming hierarchies automatically consistent, like we do in basically all other metamodels? Actually, please have a look at the `buildQualifiedName()` method of the `ServiceAspect`concept in this metamodel to see an example on how to implement `buildQualifiedName()`.